.github/workflows/update-banners.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Banners
+
+on: 
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  update-banners:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: actions/setup-node@v2.2.0
+        with:
+          node-version: '10.x'
```

Comment:
10 is EOL now

Reply:
Ahh, should I update the `.nvmrc` file then? I set this to 10 because of that file

Reply:
I think the .nvmrc change should be handled in a separate PR

Reply:
> I think the .nvmrc change should be handled in a separate PR

What version should it be bumped to?

Reply:
latest LTS sounds good to me

Reply:
https://github.com/nodejs/nodejs.dev/pull/1539 heres the PR for it